File: contest1/khoangcachnhohonk.cpp
#include <bits/stdc++.h>
using namespace std;
 
int countPairs(int a[], int n, int k)
{
    sort(a, a + n);
    int res = 0;
    for (int i = 0; i < n; i++) {

        int j = i+1;
        while (j < n && a[j] - a[i] < k) {
            res++;
            j++;
        }
    }
    return res;
}

int main()
{
	int test;
	cin >> test;
	while (test) {
		int n,k;
		cin >> n >> k;
		int a[n];
		for (int i = 0; i < n; i++) {
			cin >> a[i];
		}
    	cout << countPairs(a, n, k) << endl;
    	test--;
	}
    return 0;
}

File: contest1/khoangcachnhohonk.py
def countPairs(a, n, k):
     
    a.sort()
    res = 0
    for i in range(n):
        j = i+1
        while (j < n and a[j] - a[i] < k):
            res += 1
            j += 1
    return res
 
# Driver code
a = [1, 10, 4, 2]
k = 3
n = len(a)
print(countPairs(a, n, k), end = "")

if __name__ == '__main__':

    test = int(input())
    while test:
        a = int(input())
        b = int(input())
        print(lcm(a, b))
        print("\n")
        test-=1